src/pages/[marketplace]/deals/index.tsx

```diff
@@ -140,11 +140,32 @@ const Deals: NextPageWithLayout = () => {
 		[marketplace, publicKey]
 	);
 
+	const hasActiveCredixPass = (credixPass: CredixPass) => credixPass && credixPass.isActive;
```

Comment:
imo this function is a bit redundant. also the typing doesn't make sense here because according to the signature it's never null or undefined so the check isn't necessary then